docs/Developers/Development_Setup.rst

```diff
@@ -34,6 +34,24 @@ https://github.com/coala/coala-bears
 Installing from Git
 -------------------
 
+NOTE:
+In order to configure the environment according to python 3.
+If you have both python 3 and python 2 installed check this out first.
+
+::
+
+    $ virtualenv venv -p /usr/bin/python3
```

Comment:
nah amend the same commit

also I'd go with the first one because it is easy to type